usr/lib/common/h_extern.h

```diff
@@ -1337,6 +1337,16 @@ CK_RV aes_ctr_decrypt_update(STDLL_TokData_t *tokdata, SESSION *sess,
                              CK_BYTE *in_data, CK_ULONG in_data_len,
                              CK_BYTE *out_data, CK_ULONG *out_data_len);
 
+CK_RV aes_xts_encrypt_update(STDLL_TokData_t *tokdata, SESSION *sess,
+                             CK_BBOOL length_only, ENCR_DECR_CONTEXT *context,
+                             CK_BYTE *in_data, CK_ULONG in_data_len,
```

Comment:
const CK_BYTE *in_data   ?

Reply:
Hm yes, should be const indeed. The problem is that the caller (encr_mgr_encrypt_update) does not define in_data as const (although it should be const there, too). 
I leave it as is for this PR, but I put it on my TODP list to update all places where arguments could be const.